en-US/Design.xml

```diff
@@ -743,17 +743,27 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
 							In examples or scenarios, you can use a person's name and then use a gender-specific pronoun to refer to that name. Vary the use of proper names in documentation. Use names that represent various ethnic backgrounds, genders, and locations.
 						</para>
 
+					</listitem>
+					 <listitem>
+						<para>
+							Include a diverse set of names in your examples to reflect the diversity of the real world.
+              For example, use male, female, and culturally diverse names that suggest various backgrounds in examples to avoid implying that only certain groups have specific skills.
+						</para>
+
 					</listitem>
 					 <listitem>
 						<para>
 							Do not use copyrighted fictional characters in examples, and do not use real people.
 						</para>
 
 					</listitem>
+
 					 <listitem>
 						<para>
-							Include a diverse set of names in your examples to reflect the diversity of the real world.
-              For example, use male, female, and culturally diverse names that suggest various backgrounds in examples to avoid implying that only certain groups have specific skills.
+							When choosing names, also consider how those names might appear in email addresses, usernames, and similar contexts.
+							An example that works well might be "John Smith", with an email address of <email>jsmith@example.com</email>.
+							However, for the name "Brian Smith", a corresponding email address of <email>bstrong@example.com</email> might not work so well (when read out, it sounds like "be strong").
```

Comment:
I kind of want to know why `bstrong@` doesn't work, so that we can explain this more clearly. Is it too informal? I understand that in other cases, such abbreviations could result in inappropriate meanings -- but I'm curious about why `bstrong` doesn't work. This might be something for us to clarify.

Reply:
I don't mean "bstrong" specifically ("Be strong!") but rather be aware of how names might result in odd emails, usernames, or whatever. E.g., if you chose "Tyroner Osser" as a name the result would be not great. Long story short: be aware.